fix: move documentation to READMEs of packages (#61)
open-wc/open-wc: packages/testing-helpers/README.md

# Testing Helpers

> Part of Open Web Component Recommendation [open-wc](https://github.com/open-wc/open-wc/)

> We want to provide a good set of default on how to vasilitate your web component.

Reviewer: ```suggestion
We want to provide a good set of defaults on how to facilitate your web component.
```

[![CircleCI](https://circleci.com/gh/open-wc/open-wc.svg?style=shield)](https://circleci.com/gh/open-wc/open-wc)
[![BrowserStack Status](https://www.browserstack.com/automate/badge.svg?badge_key=M2UrSFVRang2OWNuZXlWSlhVc3FUVlJtTDkxMnp6eGFDb2pNakl4bGxnbz0tLUE5RjhCU0NUT1ZWa0NuQ3MySFFWWnc9PQ==--86f7fac07cdbd01dd2b26ae84dc6c8ca49e45b50)](https://www.browserstack.com/automate/public-build/M2UrSFVRang2OWNuZXlWSlhVc3FUVlJtTDkxMnp6eGFDb2pNakl4bGxnbz0tLUE5RjhCU0NUT1ZWa0NuQ3MySFFWWnc9PQ==--86f7fac07cdbd01dd2b26ae84dc6c8ca49e45b50)
[![Renovate enabled](https://img.shields.io/badge/renovate-enabled-brightgreen.svg)](https://renovatebot.com/)

> In order to efficient test webcomponent you will need some helpers to register and instanciate them.

Reviewer: ```suggestion
In order to efficient test webcomponent you will need some helpers to register and instantiate them.
```

::: tip Info
This is part of the default [open-wc](https://open-wc.org/) recommendation
:::

## Test a custom element
```js
import { fixture } from '@open-wc/testing-helpers';

> it('can instanciate an element', async () => {

Reviewer: ```suggestion
it('can instantiate an element', async () => {
```

  const el = await fixture('<my-el foo="bar"></my-el>');
  expect(el.getAttribute('foo')).to.equal('bar');
}
```

## Test a custom element with properties
```js
import { html, litFixture } from '@open-wc/testing-helpers';

> it('can instanciate an element with properties', async () => {

Reviewer: ```suggestion
it('can instantiate an element with properties', async () => {
```
